composer.json

```diff
@@ -14,10 +14,14 @@
         "php": ">=5.5"
     },
     "require-dev": {
-        "phpunit/phpunit": ">4.8",
-        "phpdocumentor/phpdocumentor": ">=2.9",
+        "jakoch/phantomjs-installer": "^2.1",
+        "phpunit/phpunit": "=4.8",
```

Comment:
yeh. I think I changed the phpunit version when I was trying to get some selenium tests to work. Given we are not using selenium, will revert that.